eng/versioning/version_client.txt

```diff
@@ -61,18 +61,18 @@ com.azure:azure-aot-graalvm-support-netty;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-aot-graalvm-samples;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-aot-graalvm-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-code-customization-parent;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-communication-chat;1.3.10;1.4.0-beta.1
+com.azure:azure-communication-chat;1.3.11;1.4.0-beta.1
 com.azure:azure-communication-callingserver;1.0.0-beta.4;1.0.0-beta.5
-com.azure:azure-communication-callautomation;1.0.2;1.1.0-beta.1
-com.azure:azure-communication-common;1.2.10;2.0.0-beta.2
+com.azure:azure-communication-callautomation;1.0.3;1.1.0-beta.1
+com.azure:azure-communication-common;1.2.11;2.0.0-beta.2
 com.azure:azure-communication-common-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-communication-sms;1.1.15;1.2.0-beta.1
-com.azure:azure-communication-identity;1.4.8;1.5.0-beta.1
-com.azure:azure-communication-phonenumbers;1.1.4;1.2.0-beta.1
-com.azure:azure-communication-networktraversal;1.1.0-beta.2;1.1.0-beta.3
+com.azure:azure-communication-sms;1.1.16;1.2.0-beta.1
+com.azure:azure-communication-identity;1.4.9;1.5.0-beta.1
+com.azure:azure-communication-phonenumbers;1.1.5;1.2.0-beta.1
+com.azure:azure-communication-networktraversal;1.0.0;1.1.0-beta.3
```

Comment:
It should be the latest GA version.

Reply:
https://mvnrepository.com/artifact/com.azure/azure-communication-networktraversal

Reply:
This library is not a part of the BOM, so we can ignore it.

Reply:
should we remove it? or revert it back?  or keep as it is?

Reply:
talked offline. Wil revert it.